crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1464,4 +1501,43 @@ mod tests {
             ]
         );
     }
+
```

Comment:
Can you add a failing test for when the resources are conflicting?

Reply:
Could you elaborate, please?

Reply:
As I suggested, what happens if we ask for `ResMut<A>` twice?